wp-content/mu-plugins/pantheon/pantheon-updates.php

```diff
@@ -45,7 +88,7 @@ function _pantheon_upstream_update_notice() {
 	$update_type = 'new WordPress version';
 	$update_help = 'If you need help, open a support chat on Pantheon.';
 	$data = _pantheon_curl_cached( 'https://api.live.getpantheon.com/sites/self/code-upstream-updates' );
```

Comment:
Is this call to _pantheon_curl_cached superfluous? `$data` appears unused, and this same call is made again in _pantheon_is_default_upstream().

Reply:
Looks like it, left over from use in `if` below. I'll take it out.